src/api/nodeRpcCall.js

```diff
@@ -1776,6 +1776,41 @@ const congressRepealLegislation = async (tier, index, walletAddress, callback) =
   });
 }
 
+const getTreasurySpendProposals = async () => {
+  const api = await getApi();
+  return await api.derive.treasury.proposals();
+}
+
+const getTreasurySpendPeriod = async () => {
+  const api = await getApi();
+  return api.consts.treasury.spendPeriod;
+}
+
+const getTreasuryBudget = async () => {
+  const api = await getApi();
+  const account = "5EYCAe5ijiYfyeZ2JJCGq56LmPyNRAKzpG4QkoQkkQNB5e6Z";
```

Comment:
can we use in here addres from https://github.com/liberland/liberland_frontend/pull/125 ?

Reply:
1. #125 is not merged yet
2. it would need to be refactored significantly to be reusable in different places. Something that would allow me to `dispatch(officesActions.getGovFinances.call())` and then `useSelector(officesSelectors.govFinances)` or smth.

So I think it's out of the scope for this PR.

Reply:
buuuut it should be not hardcoded or at least stored in a variable, when we deploy to prod